module/VuFind/src/VuFind/Log/Handler/StreamHandler.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/**
+ * Stream handler
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2025.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
```

Comment:
We recently had a PR to update this outdated address in all of our license headers. I've added a test (not yet merged) in #4675 to detect this problem in new files, since it's likely it will keep getting reintroduced as in-progress pull requests with new files are completed. Do you mind adjusting all of the "51 Franklin Street" comments in this PR to the new standard? You can see the exact changes in some of the files fixed as part of #4675.